glean-core/ios/Glean/Metrics/TimingDistributionMetric.swift

```diff
@@ -149,4 +149,25 @@ public class TimingDistributionMetricType {
 
         return DistributionData(fromJson: json)!
     }
+
+    /// Returns the number of errors recorded for the given metric.
```

Comment:
I'm pretty sure there's a way to reduce code duplication using `protocol` (similar to Android `interface`), but that's just tech debt and is out of scope for this PR.  

Maybe we should file a bug to reduce duplication in iOS specifically?  This could even be a "Good First Bug" or something.

Reply:
We have the same issue on Android. I'd rather see us come up with an idea for both platforms (and future ones).

Unfortunately a protocol/interface alone is not the solution. We still need to call individually different functions per metric type.

Reply:
This was just something that occurred to me as I was looking at the same function added so many times.  

I agree, if we can find a solution for all platforms, that would be ideal.